cluster-autoscaler/core/utils.go

```diff
@@ -106,17 +108,33 @@ func (podMap podSchedulableMap) set(pod *apiv1.Pod, err *simulator.PredicateErro
 	})
 }
 
-// FilterOutSchedulable checks whether pods from <unschedulableCandidates> marked as unschedulable
+// filterOutSchedulable checks whether pods from <unschedulableCandidates> marked as unschedulable
 // by Scheduler actually can't be scheduled on any node and filter out the ones that can.
 // It takes into account pods that are bound to node and will be scheduled after lower priority pod preemption.
-func FilterOutSchedulable(unschedulableCandidates []*apiv1.Pod, nodes []*apiv1.Node, allScheduled []*apiv1.Pod, podsWaitingForLowerPriorityPreemption []*apiv1.Pod,
-	predicateChecker *simulator.PredicateChecker, expendablePodsPriorityCutoff int) []*apiv1.Pod {
-	unschedulablePods := []*apiv1.Pod{}
+// TODO: filterOutSchedulable does too much to be a util func. Should be made a unexported method of StaticAutoScaler.
+// TODO: Also has too many parameters
```

Comment:
I want to do this in my next PR. Also, is there a reason why so many functions are exported in this package? I tried to check if they are being used outside the package and couldn't find usages. Maybe, I'm missing something?